src/BenchmarksApps/TodosApi/TodoApi.cs

```diff
@@ -94,6 +94,7 @@ UPDATE Todos
 }
 
 [JsonSerializable(typeof(Todo))]
+[JsonSerializable(typeof(bool?))]
```

Comment:
We call into STJ to generate schemas for all types, including simple parameters bound directly from the route (like `bool? isCompleted`). There's some scenarios where this means that users need to add additional JsonSerializable attributes for types that haven't already been captured.